docs/UserGuide.md

```diff
@@ -111,13 +127,21 @@ Examples:
 * `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01 c/948372948`
 * `add n/Betsy Crowe t/primary c/83927482 e/betsycrowe@example.com a/Downtown p/1234567 t/primary3`
 
+<div markdown="span" class="alert alert-warning">:exclamation:
+Caution: </div> 
+
+* In order to avoid potential confusion, we do not allow any student to have the same name regardless of the letter cases (e.g. Harry, HARRY, harry are deemed as same name). If you wish to add in a student who has the same name as an existing student in the student list, consider append a index number after the student's name to distinguish them.
+  e.g. Emily and Emily 2 are acceptable names.
+* We allow `PHONE_NUMBER` and `CONTACT_PARENT` to be the same as not all student have their own phone.
```

Comment:
I think this can be moved to the 'Tip' sections above instead as I feel like it isn't really a caution (Can change according to my PR #169 and use that instead) 😄

Reply:
Sure! Thank you for your suggestion.